src/calculateRentalCost.js

```diff
@@ -29,6 +29,21 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const pricePerDay = 40;
+  const smallDiscount = 20;
+  const basicDiscount = 50;
```

Comment:
I would change the naming here, it's not clear that the basic discount is bigger than small